community/blog/2020-05-New-authorization-and-enhanced-logging-in-WebApi-v1.0.1/README.md

```diff
@@ -0,0 +1,76 @@
+# New authorization & enhanced logging in WebApi v1.0
+
+With the release of version [Arcus WebApi v1.0](https://github.com/arcus-azure/arcus.webapi/releases/tag/v1.0.1), the logging functionality is greatly enhanced for better tracing in your current and future applications.
```

Comment:
```suggestion
We've shipped [Arcus Web API v1.0](https://github.com/arcus-azure/arcus.webapi/releases/tag/v1.0.1) which brings a couple of new features:
- Out-of-the-box Request tracking
- Making it easier to use HTTP correlation
- JWT Authorization
- Simplified NuGet package structure
```

Reply:
Maybe also good to mention https://github.com/arcus-azure/arcus.webapi/issues/158